ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3410,8 +3447,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-external-lb-pre
-- as: vsphere-ipi-ovn-dualstack-f14
-  cron: 43 10 8,23 * *
+- as: vsphere-ipi-external-lb-pre-f28-destructive
+  cron: 21 5 13 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@amd64 and @vsphere-ipi and @network-ovnkubernetes and not @fips
+        and @destructive'
+      TAG_VERSION: '@4.14'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```

Comment:
Wrong `TEST_FILTERS`?

Reply:
Maybe I make a mistake, is this `TEST_FILTERS` only apply to disconnected job?

Reply:
I think the `~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&` is only used for disconnected install.

Reply:
If so, I'll remove it from other connected job too.

Reply:
For a common connected install, I think we do not need to set it.